drivers/modem/gsm_ppp.c

```diff
@@ -103,6 +104,7 @@ NET_BUF_POOL_DEFINE(gsm_recv_pool, GSM_RECV_MAX_BUF, GSM_RECV_BUF_SIZE,
 		    0, NULL);
 K_KERNEL_STACK_DEFINE(gsm_rx_stack, GSM_RX_STACK_SIZE);
 K_THREAD_STACK_DEFINE(gsm_workq_stack, GSM_WORKQ_STACK_SIZE);
+static struct net_mgmt_event_callback gsm_mgmt_cb;
```

Comment:
Could you move it into `struct gsm_modem`?

Reply:
Fixed